x-pack/plugins/reporting/server/export_types/csv/create_job.ts

```diff
@@ -16,9 +15,7 @@ import {
 
 export const createJobFnFactory: CreateJobFnFactory<
   CreateJobFn<JobParamsDeprecatedCSV, TaskPayloadDeprecatedCSV>
-> = function createJobFactoryFn(reporting, parentLogger) {
-  const logger = parentLogger.clone([CSV_JOB_TYPE_DEPRECATED, 'create-job']);
-
+> = function createJobFactoryFn(reporting, logger) {
```

Comment:
There are some unrelated logging cleanups: the enqueue_job function has been rewritten and it now provides a more decorated logger to this function, so there can be less `parentLogger.clone()` in various files.